test/clear.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+require('assert');
+const should = require('should');
+const cash = require('../dist/index.js');
+
+describe('clear', function () {
+  it('should exist and be a function', function () {
+    should.exist(cash.clear);
+  });
+
+  it('should clear the terminal screen', function () {
```

Comment:
Good point! Hard to test.

Just change this to a `(function(){ /* clear goes here */}).should.not.throw();` and I think we'll be good.